benchmarks/dsmsynchbench.c

```diff
@@ -49,17 +52,15 @@ int main(int argc, char *argv[]) {
     object->state_f = 1.0;
     DSMSynchStructInit(object_combiner, bench_args.nthreads);
 
-    BarrierInit(&bar, bench_args.nthreads);
+    BarrierSet(&bar, bench_args.nthreads);
     StartThreadsN(bench_args.nthreads, Execute, bench_args.fibers_per_thread);
     JoinThreadsN(bench_args.nthreads - 1);
-    d2 = getTimeMillis();
 
     printf("time: %d (ms)\tthroughput: %.2f (millions ops/sec)\t", (int)(d2 - d1), bench_args.runs * bench_args.nthreads / (1000.0 * (d2 - d1)));
     printStats(bench_args.nthreads, bench_args.total_runs);
 
 #ifdef DEBUG
-    fprintf(stderr, "DEBUG: object state: %f\n", object->state_f);
-    fprintf(stderr, "DEBUG: object counter: %d\n", object_combiner->counter);
+    fprintf(stderr, "DEBUG: Object state: %ld\n", object_combiner->counter);
```

Comment:
In other places you write: "DEBUG: Object state counter:"
Maybe give it another short check round to verify that DEBUG messages are valid

Reply:
I'm trying to unify the DEBUG output to "Object state" everywhere, in case we want to develop a script to automatically verify these numbers.